celery/schedules.py

```diff
@@ -700,8 +715,8 @@ class solar(BaseSchedule):
     Arguments:
         event (str): Solar event that triggers this task.
             See note for available values.
-        lat (int): The latitude of the observer.
-        lon (int): The longitude of the observer.
+        lat (Union[int, float]): The latitude of the observer.
```

Comment:
```suggestion
        lat (Union[float]): The latitude of the observer.
```

Reply:
You can use just float no need for Union

Reply:
Fixed!